### ISSUE TITLE
FCP "Language" menu is only visible when FCP is not the active application

### ISSUE THREAD

**Issue by randomeizer** [fcpxhacks/fcpxhacks#2988]
I feel like it should be the opposite? Only visible when FCP is the active app?

**Comment by latenitefilms** [comment]
I think the main reason for that was that most people very rarely change the FCPX language whilst we're using it - they change it before they launch it. What we should probably do is have it available both when FCPX is not active, and when it is active.

**Comment by randomeizer** [comment]
Yeah, possibly. It was surprising to me that it wasn't visible when FCP was selected though.

**Comment by randomeizer** [comment]
Also, by having "Final Cut Pro" listed when not actually in FCP, it may be confusing for people, such as in #2986.
